plugin/src/main/java/io/jenkins/plugins/analysis/warnings/QtTranslation.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.QtTranslationParser;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import org.jenkinsci.Symbol;
+
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+
+public class QtTranslation extends ReportScanningTool {
+    private static final long serialVersionUID = 1L;
+    private static final String ID = "Qt-Translation";
+
+    /**
+     * Creates a new instance of {@link QtTranslation}.
+     */
+    @DataBoundConstructor
+    public QtTranslation() {
+        super();
+        // empty constructor required for stapler
+    }
+
+    @Override
+    public IssueParser createParser() {
+        return new QtTranslationParser();
+    }
+
+    /** Descriptor for this static analysis tool. */
+    @Symbol("Qt-Translation")
+    @Extension
+    public static class Descriptor extends ReportScanningToolDescriptor {
+        /** Creates the descriptor instance. */
+        public Descriptor() {
+            super(ID);
+        }
+
+        @Override
+        public String getDisplayName() {
+            return Messages.Warnings_QtTranslation_ParserName();
+        }
+
+        @Override
+        public String getHelp() {
+            return "Reads translation files of Qt, which are created by \"lupdate\" or \"Linguist\".";
+        }
+
+        @Override
+        public boolean canScanConsoleLog() {
+            return false;
+        }
+
+        @Override
+        public String getUrl() {
+            return "https://www.qt.io";
+        }
+
+        @Override
+        public boolean isPostProcessingEnabled() {
+            return false;
```

Comment:
When you disable that feature then the source files will not be copied to and accessible from Jenkins controller. This makes sense in cases where the file of the issue is not relevant.

Reply:
You are right, this was still copy and paste from the trivy PR.